extra/3-magic-8-ball.js

```diff
@@ -58,9 +99,21 @@ function shakeBall() {
 
   This function should expect to be called with any value which was returned by the shakeBall function.
 */
+
 function checkAnswer(answer) {
   //Write your code in here
+
+  if (veryPositive.includes(answer)) {
+    return "very positive";
+  } else if (positive.includes(answer)) {
+    return "positive";
+  } else if (negative.includes(answer)) {
+    return "negative";
+  } else if (veryNegative.includes(answer)) {
+    return "very negative";
```

Comment:
nice - this is a good use case for the array method "includes"

Reply:
Thank you Paul. This was refactored with the help of Guardians.